tools/scripts/acr-scan.ps1

```diff
@@ -0,0 +1,97 @@
+<#
+ .SYNOPSIS
+    Returns relevant scan results for a ACR registry.
+
+ .DESCRIPTION
+    Returns relevant scan results for a ACR registry
+    which can be further processed by another script
+    or converted to json by piping it to ConvertTo-Json
+    commandlet. The script requires az (AzureCLI) to 
+    be installed and you must be logged in (az login)
+
+ .PARAMETER Registry
+    The name of the registry
+
+ .PARAMETER Subscription
+    The subscription to use - otherwise uses default.
+
+ .PARAMETER All
+    Include also vulnerabilities that are not patchable.
+#>
+
+Param(
+    [string] $Registry = "industrialiotprod",
+    [string] $Subscription = "IOT_GERMANY",
+    [switch] $All
```

Comment:
not sure if we should expose this information

Reply:
It is in several other scripts already.  We should move all these scripts to another repo eventually.